Update bridge config for latest rpc changes

phahulin opened the pull request:
Update bridge config file to remove fields that are no longer required,example:

```
keystore = "keys"

[home]
account = "0x006E27B6A72E1f34C626762F3C4761547Aff1421"
required_confirmations = 0
poll_interval = 1
request_timeout = 5
rpc_host = "https://sokol.poa.network"
rpc_port = 443
password = "home-password.txt"

[foreign]
account = "0x006E27B6A72E1f34C626762F3C4761547Aff1421"
required_confirmations = 0
poll_interval = 1
request_timeout = 5
rpc_host = "https://kovan.infura.io/mew"
rpc_port = 443
password = "foreign-password.txt"

[authorities]
accounts = [
  "0xe388c256c31ff953343ce81160b5ca9b564c1a32",
  "0xbbefa45868cb8d6d445b78592175e4f1049570b0",
  "0x2822e60af67f3713e696d80281e7e62fc6619dfe"
]
required_signatures = 1

[transactions]
deposit_relay = { gas = 3000000, gas_price = 1000000000 }
withdraw_relay = { gas = 3000000, gas_price = 1000000000 }
withdraw_confirm = { gas = 3000000, gas_price = 1000000000 }
```

@akolotov  could you please take a look, I might have missed something

akolotov commented:
1. https://github.com/poanetwork/deployment-bridge/pull/7/files#diff-0cad1e37d72d823c8457513cb242c578R73 line and the next one needs to be removed since are not used by new bridge.
2 https://github.com/poanetwork/deployment-bridge/pull/7/files#diff-0cad1e37d72d823c8457513cb242c578R100 - is it worth to mention about remote logging facility somewhere?

phahulin commented on upgradable-wo-parity/roles/bridge/defaults/main.yml:
```diff
@@ -27,11 +30,16 @@ bridge_bin_sha256: "59405959eeb8b3095543b3e385b8cbb0bf0215a6fc1a40bce65ae9c71ede
 bridge_home_password_file: "home-password.txt"
 bridge_foreign_password_file: "foreign-password.txt"
+
+bridge_home_poll_interval: 1
```
Agree, but Id set it to 4 sec for POA, so that with network latency and processing time it would be approx 1 block per poll. 3 for kovan is fine for the same reason.

phahulin commented:
@akolotov are you sure it's https://github.com/poanetwork/deployment-bridge/pull/7/files#diff-0cad1e37d72d823c8457513cb242c578R73 line (`foreign_deploy`) and **next one** (`checked_deposit_relay`), not previous one (`home_deploy`)?

akolotov commented:
@phahulin yes. you are right.

phahulin commented:
@akolotov I will merge this addressing your comments, leaving foreign_deploy and home_deploy fields for now, until https://github.com/poanetwork/poa-bridge/issues/67 is closed
